boot/espressif/hal/include/esp32h2/sdkconfig.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define BOOTLOADER_BUILD 1
+#define CONFIG_IDF_FIRMWARE_CHIP_ID 0x0010
+#define CONFIG_IDF_TARGET_ESP32H2 1
+#define CONFIG_ESP32H2_REV_MIN_0 1
+#define CONFIG_ESP32H2_REV_MIN_FULL 0
+#define CONFIG_ESP_REV_MIN_FULL CONFIG_ESP32H2_REV_MIN_FULL
+#define CONFIG_ESP32H2_REV_MIN 0
+#define CONFIG_ESP32H2_REV_MAX_FULL 99
+#define CONFIG_ESP_REV_MAX_FULL CONFIG_ESP32H2_REV_MAX_FULL
+#define CONFIG_IDF_TARGET_ARCH_RISCV 1
+#define CONFIG_MMU_PAGE_SIZE 0x10000
+#define SOC_MMU_PAGE_SIZE CONFIG_MMU_PAGE_SIZE /* from soc/CMakeLists */
+#define CONFIG_XTAL_FREQ 40
```

Comment:
Please, confirm if ESP32-H2 supports a 40MHz XTAL. I recall it only supported 32MHz XTAL.

https://github.com/espressif/esp-idf/blob/9a1cc59338aa151bc6333f85c8169d36a478fe2a/components/soc/esp32h2/include/soc/Kconfig.soc_caps.in#L190-L192